### ISSUE TITLE
Crear rutas privadas

### ISSUE THREAD

**Issue by 9gustin** [frontendcafe/air-flashcards#16]
Crear la logica para hacer que una ruta sea privada

**Comment by kevin-dev71** [comment]
tengo una propuesta para eso utilizando `Guards`.. puedes ver este comentario que deje en la issue  https://github.com/frontendcafe/air-flashcards/issues/19#issuecomment-1210157155

Para manejar el usuario no crees q sea mejor implementar un context que haga el `getAuth()` de firebase y asi lo accedemos desde un context?

**Comment by AnaliaAlvarez** [comment]
@dafnecaneda este issue ya debería estar resuelto ya que reutilicé el componente `AuthGuard` propuesto por @kevin-dev71 para las rutas no accesibles para el usuario al estar logueado. El PR #50 ya esta mergeado.

**Comment by tomi-mercado** [comment]
Agree. @AnaliaAlvarez , asignátelo y pasalo a done porfis. Lo mejor es esto hacerlo y tener claro en qué issue estamos trabajando antes de empezar a codear.
